IM_calculation/scripts/calculate_ims.py

```diff
@@ -206,7 +206,7 @@ def main():
 
     # Create output dir
     utils.setup_dir(args.output_path)
-    if not args.simple_output:
+    if not args.simple_output and args.advanced_ims is None:
```

Comment:
Is the default value for nargs="+" None? Just checking.

Reply:
yes, it will be None not a empty list or such.

Reply:
I think that this isn't a robust solution to your problem. Expecting nothing else to be in the folder isn't a great solution. Could you get the observed station list from the input waveforms? or potentially write a list of stations as part of IM_calc?

Reply:
the current way of getting waveforms is from scanning the entire folder for *.000, instead of reading a file that contains a list. 
so we can potenailly generate a temporary file that contains that list and use that as the input to check_status?

Reply:
Yep. That sounds good. A permanent file rather than a temporary one but sure.

Reply:
https://github.com/ucgmsim/slurm_gm_workflow/pull/386/commits/654a5280f397e63922fecb6713abf7e38cdd8b9b
i've update the script to just use the get_bbseis function from im_calc.
so on files needs to be saved, as its using the same logic